src/reactphp/src/ServerFactory.php

```diff
@@ -36,7 +36,7 @@ public function createServer(RequestHandlerInterface $requestHandler): LoopInter
         $loop->addSignal(SIGTERM, function (int $signal) {
             exit(128 + $signal);
         });
-        $loop->addSignal(SIGKILL, function (int $signal) {
+        $loop->addSignal(SIGHUP, function (int $signal) {
```

Comment:
I'm sadly not enough familiar here with this kind of signals. @Nyholm maybe you can have a look at this.

I did just stumble over the examples where the signal is also removed: https://github.com/reactphp/event-loop/blob/21914ff1db4260ca4c81e338c7e235a4df4ef94b/examples/04-signals.php#L14 is that also simething we are required but maybe not required as we are exiting here.

Reply:
yeah, I thought it might be removable entirely.